tests/tc_ufcs_string_and_string_literal_completion/file.d

```diff
@@ -0,0 +1,6 @@
+void ufcsString(string input){}
+void ufcsStringBar(string input){}
+void testUfcs(string x){
+	"foo".
+	x.
```

Comment:
the dot here makes the x not work properly / the x should never match anything, since it's `"foo".x.` (which is an unknown property / not an ufcs method)

However since the test actually matches something here, this indicates that the UFCS code is somehow broken here.

Reply:
Ah I see, that's right nicely spottet.